Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/kimai/kimai?pullRequest=4946) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/kimai/kimai?pullRequest=4946) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/kimai/kimai?pullRequest=4946) it.</sub>

weblate committed:
Translated using Weblate (Danish)

Currently translated at 78.7% (344 of 437 strings)

Translated using Weblate (French)

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (French)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (French)

Currently translated at 98.7% (77 of 78 strings)

Translated using Weblate (French)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (French)

Currently translated at 98.8% (432 of 437 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 78.7% (344 of 437 strings)

Translated using Weblate (Arabic)

Currently translated at 92.4% (404 of 437 strings)

Translated using Weblate (Arabic)

Currently translated at 91.9% (402 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 64.3% (281 of 437 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 87.1% (68 of 78 strings)

Translated using Weblate (Danish)

Currently translated at 64.3% (281 of 437 strings)

Translated using Weblate (Czech)

Currently translated at 99.7% (436 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (French)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (French)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Danish)

Currently translated at 83.3% (10 of 12 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (French)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (8 of 8 strings)

Translated using Weblate (Danish)

Currently translated at 62.8% (49 of 78 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (31 of 31 strings)

Translated using Weblate (Danish)

Currently translated at 97.3% (37 of 38 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 52.6% (230 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 11.1% (3 of 27 strings)

Translated using Weblate (Danish)

Currently translated at 44.4% (4 of 9 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Danish)

Currently translated at 77.7% (7 of 9 strings)

Translated using Weblate (Danish)

Currently translated at 69.2% (9 of 13 strings)

Translated using Weblate (Danish)

Currently translated at 69.2% (9 of 13 strings)

Translated using Weblate (Danish)

Currently translated at 58.3% (7 of 12 strings)

Translated using Weblate (Danish)

Currently translated at 33.3% (4 of 12 strings)

Translated using Weblate (Danish)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Danish)

Currently translated at 64.5% (20 of 31 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Danish)

Currently translated at 52.1% (228 of 437 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (38 of 38 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (437 of 437 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (437 of 437 strings)

Co-authored-by: Ahmad Dakhlallah <ahmad@linuxarabia.co>
Co-authored-by: Bent Haar <BentHaar@users.noreply.hosted.weblate.org>
Co-authored-by: Carlos Carreras <mytriponlinux@gmail.com>
Co-authored-by: David Bauer <dbs23.dbs23@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Jose Delvani <delvani.eletricista@gmail.com>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Ldm Public <ldmpub@gmail.com>
Co-authored-by: Mads R. Andersen <madsdk_789@hotmail.com>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Mikkel Ricky <rimi@aarhus.dk>
Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Quiwy <github@quiwy.ninja>
Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: geekom13 <geekom13@proton.me>
Co-authored-by: lilosti <lilosti@aarhus.dk>
Co-authored-by: ssantos <ssantos@web.de>
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/export/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/reporting/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/fr/
Translation: Kimai - Time tracking/Actions (Dropdowns and Toolbars)
Translation: Kimai - Time tracking/Date-Picker
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Export
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Invoice-Templates
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Plugins
Translation: Kimai - Time tracking/Reporting
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/Validations
Translation: Kimai - Time tracking/Wizard
